padinfo/view/series_scroll.py

```diff
@@ -124,6 +145,63 @@ async def query_from_ims(dgcog, ims) -> List[List["MonsterModel"]]:
         paginated_monsters = await SeriesScrollViewState.do_query(dgcog, series_id, rarity, query_settings.server)
         return paginated_monsters
 
+    async def decrement_page(self, dgcog, ims: dict):
```

Comment:
I don't think you need ims right? just dgcog - the other function was actually modifying in the ims, but here we're not